### ISSUE TITLE
babelParserPlugins not working.

### ISSUE THREAD

**Issue by adjenks** [HcySunYang/vuese#224]
**Describe the bug**
Setting babelParserPlugins doesn't seem to do anything.

**To Reproduce**
1. Create a vuese.config.js (or a .vueserc )
2. Fill it in with this:
```
module.exports = {
	"babelParserPlugins": [
		'optionalChaining'
	]
}
```
3. Create a vue file with some optional chaining: `var test = a?.b?.c`
4. Run vuese gen
5. > SyntaxError: This experimental syntax requires enabling the parser plugin: 'optionalChaining'

**Expected behavior**
No syntax error

**Environmental information:**
vuese/2.10.0 linux-x64 node-v14.15.4

**Additional information**
I know it's parsing the file because if I make a syntax error in `vuese.config.js` it breaks vuese.

**Comment by adjenks** [comment]
Apparently my version was out of date. I ran `vuese --version` and it told me 2.10.0, but this is the parser version?? I updated and now it's 1.4.7. Very confusing, I thought 1.4.7 must be old because I was on version 2.10.0.

You can close this ticket, but I thought you should read it first so I won't close it myself.

**Comment by IWANABETHATGUY** [comment]
try to upgrade your `@vuese/cli` to the latest `2.13.1` , i have tested  it.

**Comment by adjenks** [comment]
@IWANABETHATGUY , yes, I guess I forgot to say that it's working now. I basically uninstalled vuese and reinstalled it and it everything started working.

What I was trying to point out though, was that the output from `vuese --version` from an older version was `2.10.0` and the newer version said `1.4.7`. So it was a bit confusing because the new version had an older number, I think the old one that I had outputted the version of the cli, but the new version outputted the parser version?

Hold that though, okay, I just realised that there is `vuese` and also `@vuese/cli`. So I uninstalled `vuese` and installed `@vuese/cli` and it also works but now the version is reporting like so:
>$ vuese --version
>vuese/2.14.3

I guess I was just supposed to be using @vuese/cli. So how are they different? Should anyone be installing `vuese` and not `@vuese/cli`? The version number seems out of date on `vuese`.
